### ISSUE TITLE
"Create procedure": Handle return value type and length

### ISSUE THREAD

**Issue by NKarolak** [DavidFeldhoff/al-codeactions#41]
```
    procedure MyFunction()
    var      
        MyCode5Variable: Code[5];
    begin
        MyCode5Variable := MyNewFunction('CCCCC');     // missing
    end;
```
"Create procedure":
Currently:
```
 local procedure MyNewFunction(arg: Text): Variant
    begin
        Error('Procedure not implemented.');
    end;
```

Expected:
```
 local procedure MyNewFunction(arg: Text): Code[5]
    begin
        Error('Procedure not implemented.');
    end;
```
That would be great :)

**Comment by DavidFeldhoff** [comment]
Thanks :) I'll look into that and keep you posted!

**Comment by DavidFeldhoff** [comment]
I just tried it with this exact example, but I couldn't reproduce it as it recognizes the return type correctly. It does even work if I create a procedure as a parameter of another procedure call (so that it takes the type of the parameter) or something like that.
What am I doing wrong? :/

**Comment by NKarolak** [comment]
Interesting...
My code sample was actually a simplification, thoughclosetotheoriginal. I will try to reproduce it again tomorrow, maybe it's all about a tiny detail only. 

BTW, I often see parameter variable names with spaces = in quotation marks. 
Are there any plans to create variable names without spaces, ingeneral?

**Comment by DavidFeldhoff** [comment]
You mean that the parameters are always created without spaces and without quotation marks?
I could make it happen.. Good idea :)

**Comment by NKarolak** [comment]
Yes, I would prefer that. And Microsoft apparently too 😉

**Comment by DavidFeldhoff** [comment]
Okay, I created a separate issue for that, so that we can focus on the issue addressed here. Hope you can find the detail, so that I can fix that :)

**Comment by NKarolak** [comment]
Wow, this was tricky to find out.

Please try again using this code:

```
 procedure MyFunction()
    var
        MyCode5Variable: Code[5];
    begin
        // MyCode5Variable := 'CCCCC'; // <--- new line
        MyCode5Variable := MyNewFunction('CCCCC');     // missing
    end;
```

**Comment by DavidFeldhoff** [comment]
Oh.. Nice one 😆 
Thanks for keep trying, I'll look into that tomorrow and fix it 👍

**Comment by DavidFeldhoff** [comment]
Hi Natalie, I fixed it yesterday with v0.2.5.
Furthermore I found another issue: you opened a few days back an issue regarding usage of a table field which is an enum which I fixed. But if the enum was used directly, then it was of type variant again. This is now fixed as well.
